doc/shared_library_howto/howto.txt

```diff
@@ -0,0 +1,107 @@
+
+Compiling and Using Theano functions as shared libraries
+========================================================
+
+First one important note: this functionality is experimental, so not every
+features works, and some may not work as you expected.
```

Comment:
This would be better served with a `.. note` section to draw attention to it.

Also the text would read better as:

This functionality is experimental.  Not every feature works and some may break in unexpected ways or behave differently.